config/configuration.json

```diff
@@ -36,7 +36,7 @@
     }
   },
   "BuildingRoadIntersectionCheck": {
-    "highway.filter": "highway->motorway_link,primary_link,primary,residential,secondary_link,secondary,service,tertiary_link,tertiary,track,trunk_link,trunk,unclassified,motorway,road",
+    "highway.car.navigable": true,
```

Comment:
This could probably just be `car.navigable`

Reply:
I have updated this in the latest commit.